RunningOrder/SplitViewControllerAccessor.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  File.swift
+//  RunningOrder
+//
+//  Created by Lucas Barbero on 05/08/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import Foundation
+import Cocoa
+
+class SplitViewControllerAccessor: SplitViewControllerProtocol {
```

Comment:
WHYYYYYYYYYY ?

Reply:
why don't you conform the AppWindowController as it has already the splitviewcontroller ? Why creating a new thing in addition ?